### ISSUE TITLE
Undefined variable: user on line 115 of Import_EntryService.php

### ISSUE THREAD

**Issue by alexyoung91** [boboldehampsink/import#43]
Without the author defined in the CSV, the import fails with this error, temporarily fixed by changing

```
$element->$author = ($element->$author ? $element->$author : ($user = craft()->userSession->getUser() ? $user->id : 1));
```

to

```
$user = craft()->userSession->getUser();
$element->$author = ($element->$author ? $element->$author : ($user ? $user->id : 1));
```

**Comment by rgbjay** [comment]
This is great, fixed the issue I was facing in #42

**Comment by boboldehampsink** [comment]
Thanks for pointing this out. I've just released 0.8.18 which fixes this.
